crates/viewer/re_space_view_spatial/src/visualizers/ellipsoids.rs

```diff
@@ -100,8 +100,14 @@ impl Ellipsoids3DVisualizer {
 
                 // TODO(kpreid): subdivisions should be configurable, and possibly dynamic based on
                 // either world size or screen size (depending on application).
-                let subdivisions = 2;
-                let proc_mesh_key = proc_mesh::ProcMeshKey::Sphere { subdivisions };
+                let subdivisions = 4;
```

Comment:
```suggestion
                let subdivisions = match data.fill_mode {
                    FillMode::DenseWireframe => 2, // Don't make it too crowded - let the user see inside the mesh.
                    FillMode::Solid => 6, // Smooth, but not too CPU/GPU intensive
                    FillMode::MajorWireframe => 8, // Three smooth ellipses 
                };
```

Reply:
This kind of breaks what I see as an important consistency property: right now, you can draw two entities with different fill modes and the same data to emulate a wireframe+fill rendering. With this change, the DenseWireframe lines would be partially obscured by the more detailed Solid surface, and the MajorWireframe would also be both partially obscured and come away from the surface.

But perhaps that's not a big deal, and I do like the idea of putting in more detail when it is likely wanted.

Reply:
Good points, but the low-res equator lines looks so ugly 😬

Reply:
We've talked it through a bit internally, and we've decided that beautiful (smooth) lines is more important. We should still add a comment here for future reference though.

> you can draw two entities with different fill modes and the same data to emulate a wireframe+fill rendering

This is really the source of the problem. If we use a single entity with separate fill color and outline color, the problem goes away, and we'll get there at some point (not this PR).